crates/goose/Cargo.toml

```diff
@@ -32,4 +32,7 @@ indoc = "2.0.5"
 wiremock = "0.6.0"
 mockito = "1.2"
 tempfile = "3.8"
+tokio = { version = "1.0", features = ["full", "test-util"] }
 dotenv = "0.15"
+[features]
```

Comment:
nit: like mentioned above would maybe treat this like other integration tests in tests/